README.md

```diff
@@ -50,23 +120,24 @@ they will just retry until it does.
 - `make dependencies` to download vendor dependencies using Glide.
 - `make packages` to generate binaries for several platforms.
 
-You will find binaries in `borges_linux_amd64/borges` and `borges_darwin_amd64/borges`. 
+You will find binaries in `borges_linux_amd64/borges` and `borges_darwin_amd64/borges`.
 
 If running for the first time, you also need to add the following table to PostgreSQL:
 
 ```sql
 CREATE TABLE IF NOT EXISTS repositories (
-    id uuid PRIMARY KEY,
-    created_at timestamptz,
-    updated_at timestamptz,
-    endpoints text[],
-    status varchar(20),
-    fetched_at timestamptz,
-    fetch_error_at timestamptz,
-    last_commit_at timestamptz,
-    _references jsonb
-);
-    
+	id uuid PRIMARY KEY,
+	created_at timestamptz,
+	updated_at timestamptz,
+	endpoints text[],
+	status varchar(20),
+	fetched_at timestamptz,
+	fetch_error_at timestamptz,
+	last_commit_at timestamptz,
+	is_fork boolean,
```

Comment:
do we want to keep them all as nullable? Also, shouldn't this be a script/command/etc instead of being in the README?